src/test/finalization/state_db_tests.cpp

```diff
@@ -174,9 +173,9 @@ BOOST_AUTO_TEST_CASE(load_best_states) {
     db->LoadStatesHigherThan(59, &restored);
     BOOST_CHECK_EQUAL(restored.size(), 140);
 
-    for (auto it = restored.begin(); it != restored.end(); ++it) {
-      const CBlockIndex *index = it->first;
-      const esperanza::FinalizationState &state = it->second;
+    for (auto &it : restored) {
```

Comment:
nit: it's not iterator, I'd rename to `kv` to emphasis "key value" where `kv.first` is a key, and `kv.second` is value.

Reply:
Changed in fabef31f9727d6dfb53b58d0a500614507e3881f